spec.bs

```diff
@@ -116,6 +132,49 @@ if (document.wasDiscarded) {
 }
 </pre>
 
+Feature Policies {#feature-policies}
+=====================
+To improve CPU utilization of [=nested browsing contexts=] that are not critical to a page's usability two feature policies are defined:
```

Comment:
To clarify: in some cases, a parent page may want to use these feature policies for business logic purposes, not just CPU savings. For instance, a parent web page may want to pause the playback in an iframe w/o resorting (or not trusting to) 3p iframe messaging. This is a UX need, with CPU savings being a good side-effect.

Reply:
done